python/ray/serve/_private/client.py

```diff
@@ -416,6 +417,10 @@ def get_all_deployment_statuses(self) -> List[DeploymentStatusInfo]:
             for status_bytes in statuses_bytes
         ]
 
+    @_ensure_connected
+    def get_fastapi_docs_path(self, name: str):
+        return ray.get(self._controller.get_fastapi_docs_path.remote(name))
```

Comment:
is this purely for testing?

Reply:
instead of adding this method, can we go through whatever method the REST API calls? we should have one method that dumps the state in a format similar to the REST API output schema and validate that

Reply:
Haven't added the support for that yet, but you're right this method can be removed. I'll modify the test.